src/data/item-desc.csv

```diff
@@ -1,75 +1,75 @@
+"デイリーレースの挑戦に必要なチケット。","A ticket necessary for participating in daily races."
+"レジェンドレースの挑戦に必要なチケット。","A ticket necessary for participating in legend races."
+"トレーナー同士の友情の証。\nショップで様々なものと交換できる。","Friendship points exchangeable for items at the shop."
+"クラシック級GⅠ・ジャパンダートダービー\nを優勝した証。覚醒Lv強化で使用できる。","G1 Japan Dirt Derby victory proof.\nUsed to raise Awakening Lv."
+"ダートGⅠ・JBCスプリントを優勝した証。\n覚醒Lv強化で使用できる。","G1 JBC Sprint victory proof.\nUsed to raise Awakening Lv."
+"ダートGⅠ・JBCクラシックを優勝\nした証。覚醒Lv強化で使用できる。","G1 JBC Classic victory proof.\nUsed to raise Awakening Lv."
+"ダートGⅠ・JBCレディスクラシックを\n優勝した証。覚醒Lv強化で使用できる。","G1 JBC Ladies Classic victory proof.\nUsed to raise Awakening Lv."
+"ダートGⅠ・帝王賞を優勝した証。\n覚醒Lv強化で使用できる。","G1 Teio Sho victory proof.\nUsed to raise Awakening Lv."
+"ダートGⅠ・東京大賞典を優勝した証。\n覚醒Lv強化で使用できる。","G1 Tokyo Daishoten victory proof.\nUsed to raise Awakening Lv."
+"能力を引き出す不思議な力。\nサポートカードのLv強化で使用できる。","Support points used to increase the level of support cards."
+"サポートカードガチャが引けるチケット。\nガチャの一覧から使用できる。","A support card gacha pull ticket.\nUsable at the Gacha menu."
+"★3確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。","A ★3 guaranteed character gacha pull ticket.\nUsable at the gacha menu."
+"SSR確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。","A SSR guaranteed support card gacha pull ticket.\nUsable at the gacha menu."
+"三女神様の祝福が得られるとされている。\nショップで様々なものと交換できる。","Goddess Statues exchangeable for items at the shop."
+"ウマ娘のやる気を絶好調にする。\nチーム競技場などで使用できる。","Raises Motivation to Maximum.\n(Team Stadium etc)."
+"天候が「晴」、バ場が「良」になる。\nチーム競技場などで使用できる。","Weather turns 'Sunny' and Terrain turns 'Good'.\n(Team Stadium etc)"
+"天候が「雨」、バ場が「不良」になる。\nチーム競技場などで使用できる。","Weather turns 'Rainy' and Terrain turns 'Bad'.\n(Team Stadium etc)"
+"枠番が内枠（1～3）になる。\nチーム競技場などで使用できる。","Gate block changes to inner (1-3).\n(Team Stadium etc)"
+"枠番が外枠（6～8）になる。\nチーム競技場などで使用できる。","Gate block changes to outer (6-8).\n(Team Stadium etc)"
+"リストの中から好きな\n★3育成ウマ娘と交換できる引換券。","Exchangeable for a ★3 character from a list."
+"リストの中から好きな\nSSRサポートカードと交換できる引換券。","Exchangeable for an SSR support card from a list."
+"持つべきものはフォロー数！\nフォロー枠を1つ増やすことができる。","Increases the maximum trainer follow count by one."
+"サークルの団結の証。\nショップで様々なものと交換できる。","Circle Points exchangeable for items at the shop."
+"★2以上確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。","A ★2+ guaranteed character gacha pull ticket.\nUsable at the gacha menu."
+"SR以上確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。","A SR+ guaranteed support card gacha pull ticket.\nUsable at the gacha menu."
+"タウラス杯のエントリーに\n使用できるチケット。\n※次回開催に持ち越すことはできません。","An entry ticket for the Taurus Hai.\nDoes not carry over between race periods."
+"ジェミニ杯のエントリーに\n使用できるチケット。\n※次回開催に持ち越すことはできません。","An entry ticket for the Gemini Hai.\nDoes not carry over between race periods."
+"キャンサー杯のエントリーに\n使用できるチケット。\n※次回開催に持ち越すことはできません。","An entry ticket for the Cancer Hai.\nDoes not carry over between race periods."
+"レオ杯のエントリーに\n使用できるチケット。\n※次回開催に持ち越すことはできません。","An entry ticket for the Leo Hai.\nDoes not carry over between race periods."
```

Comment:
Have you checked if they look fine without the markup? I think it's also shown in multiple places so adjusting it to smaller size ensures it's not cut-off in more places. @robflop probably knows more.

Reply:
So far every instance I've seen in game had the markup show up as-is in text, causing cut-offs at times, so I am assuming it doesn't work (anymore?).

Reply:
I don't actually use the patch myself so I can't verify this, but from what I remember in the last PR about item descriptions, these already have problems with the mark-up. So removing it would just amplify them. Can't give specifics though, would need someone to manually check both.

Reply:
![image](https://user-images.githubusercontent.com/7202881/131830006-a9448394-c50b-4d90-9658-fb89cac15b26.png)
![image](https://user-images.githubusercontent.com/7202881/131830042-72e741cb-bad3-4c81-8731-3ffe8b41051c.png)
![image](https://user-images.githubusercontent.com/7202881/131830089-93186857-1089-44e5-b3a8-831cf213b5b8.png)

I'm actually not too sure what's going on here, lol. The markup works properly on some items, gets ignored on others, and displays as text on yet another group, seemingly randomly. Don't really have the time to check what's going on in detail now, but there's definitely something odd about this.

Reply:
![weird](https://user-images.githubusercontent.com/80930594/132092906-ba242c4f-2232-44b9-a4cd-a68f99188470.png)
Adding a "\n" at the start of </size> seems to fix the issue
In other words the markup got cut off
(<\siz
e>)
and it didn't register, it's dumb